src/components/translator/Translator.tsx

```diff
@@ -272,6 +277,11 @@ const Translator = ({ mode: initialMode }: { mode?: Mode }): React.ReactElement
 
   const onTranslate = React.useCallback(() => window.dispatchEvent(new Event(TranslateEvent)), []);
 
+  const swapLangText = () => {
```

Comment:
Let's use `React.useCallback()`.

Reply:
Let's also call this `swapLangs`. It could be used for document translation as well.